src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -317,7 +317,7 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                 files.AddRange(srcElement.Value.Trim(';').Split(';').Select(s => 
                     new ManifestFile
                     {
-                        Source = s.SafeTrim().TrimStart(slashes),
```

Comment:
It's right there in the test I changed.  The line above the one I changed tests leading a leading slash in the target.